Source/PagesController.swift

```diff
@@ -16,12 +22,27 @@ import UIKit
     }
   }
 
+  public var showBottomLine = false {
+    didSet {
+      bottomLineView.hidden = !showBottomLine
+    }
+  }
+
   lazy var pages = Array<UIViewController>()
 
   public private(set) var currentIndex = 0
 
   public var pagesDelegate: PagesControllerDelegate?
 
+  public private(set) var bottomLineView: UIView = {
+    let view = UIView(frame: CGRectZero)
```

Comment:
![Giphy](http://media0.giphy.com/media/EwX2Nby7DUpQQ/giphy.gif)
just giffed (googled - giffed") this word :D